internal/ethapi/api.coreth.go

```diff
@@ -63,6 +61,18 @@ func (s *EthereumAPI) SuggestPriceOptions(ctx context.Context) (*PriceOptions, e
 		return nil, nil
 	}
 
+	// Find min base fee based on chain config
+	// TODO: This can be removed after Fortuna is activated
+	time := s.b.CurrentHeader().Time
+	chainConfig := s.b.ChainConfig()
+	var minBaseFee int64
+	if chainConfig.IsFortuna(time) {
+		minBaseFee = acp176.MinGasPrice
+	} else {
+		minBaseFee = etna.MinBaseFee
+	}
+	bigMinBaseFee := big.NewInt(minBaseFee)
```

Comment:
nit could assign directly to a big.Int

```suggestion
	minBaseFee := new(big.Int)
	if chainConfig.IsFortuna(time) {
		minBaseFee.SetUint64(acp176.MinGasPrice)
	} else {
		minBaseFee.SetUint64(etna.MinBaseFee)
	}
```

and remove `bigMinBaseFee`